flake.nix

```
@@ -94,7 +94,8 @@
           };
 
           tailscaleAuthKeyFile = mkOption {
-            type = types.path;
+            type = types.nullOr types.path;
+            default = null;
             description = "Path to file containing the Tailscale Auth Key";
```

Since the flake is the documentation, can the description be updated to say where to look for the auth login flow.

```suggestion
            description = "Path to file containing the Tailscale Auth Key, if null check golink journal for login link to authorize against the tailnet.";
```